packages/serverless-orchestration/test/ServerlessHub.js

```diff
@@ -324,7 +324,8 @@ contract("ServerlessHub.js", function(accounts) {
           CUSTOM_NODE_URL: web3.currentProvider.host,
           POLLING_DELAY: 0,
           EMP_ADDRESS: emp.address,
-          PRICE_FEED_CONFIG: defaultUniswapPricefeedConfig
+          PRICE_FEED_CONFIG: defaultUniswapPricefeedConfig,
+          LIQUIDATOR_CONFIG: { contractVersion: "latest", contractType: "ExpiringMultiParty" }
```

Comment:
added here to make the serverless compatible with the new changes. note this is only needed within these unit tests because the contract version deployed here is slightly different due to package versioning. This simply acts to override the default version.